cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A protocol for implementing high performance clifford tableau evolutions
+ for Clifford Simulator."""
+
+from typing import Any, Iterable, TYPE_CHECKING
+
+import numpy as np
+
+from cirq.ops import common_gates
+from cirq.ops import pauli_gates
+from cirq.ops.clifford_gate import SingleQubitCliffordGate
+from cirq.protocols import has_unitary, num_qubits, unitary
+from cirq.sim.clifford.stabilizer_state_ch_form import StabilizerStateChForm
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class ActOnStabilizerCHFormArgs:
+    """State and context for an operation acting on a stabilizer state in CH
+    form. To act on this object, directly edit the `state` property, which is
+    storing the stabilizer state of the quantum system with one axis per qubit.
+    Measurements are currently not supported on this object.
+    """
+
+    def __init__(self, state: StabilizerStateChForm, axes: Iterable[int]):
+        """
```

Comment:
Please fix the docs string according to our [style guide](https://github.com/quantumlib/Cirq/blob/master/docs/dev/development.md#writing-docstrings-and-generating-documentation).

Namely: no single line intro or description here.

Reply:
Done